openmrs/omod/src/main/java/org/openmrs/projectbuendia/webservices/rest/OrderResource.java

```diff
@@ -341,6 +409,30 @@ private Order reviseOrder(Order order, SimpleObject edits) {
                     }
                     break;
 
+                case "start_millis":
```

Comment:
It looks like Ping changed the field name from "stop" to "stop_millis" here, so it's ok to remove the "stop".